libraries/standard/http/include/http_client.h

```diff
@@ -403,19 +439,44 @@ HTTPStatus_t HTTPClient_AddHeader( HTTPRequestHeaders_t * pRequestHeaders,
  * #HTTPRequestHeaders_t.pBuffer.
  *
  * For example, if requesting for the first 1kB of a file the following would be
- * written  "Range: bytes=0-1024\r\n".
+ * written  "Range: bytes=0-1024\r\n\r\n".
```

Comment:
The range is inclusive on both ends. So it would be `Range: bytes=0-1023\r\n\r\n`

Reply:
Fixed